DayrSuryan/DSEthiop3.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>ʾArgānona Māryām</title>
```

Comment:
Don't we want to change to "Harp of Mary"?

Reply:
I would be in favour of "Organ of...", which is the literatl translation and title given in the EAE. I realize that the matter of musical instruments is complicated...